modules/oc-by-example-content.adoc

```diff
@@ -1857,7 +1857,13 @@ Log in to a server
   oc login localhost:8443 --web --callback-port 8280
 
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
-  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080 --issuer-url=$ISSUER_URL
+
+  # Log in with an external OIDC
+  oc login --exec-plugin=oc-oidc --issuer-url=$ISSUER_URL --client-id=$CLI_CLIENT_ID --extra-scopes=email,profile --callback-port=8080
+
+  # Log in with an external OIDC if the external OIDC certificate is not publically trusted
+  oc login --exec-plugin=oc-oidc --issuer-url=$ISSUER_URL --client-id=$CLI_CLIENT_ID --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>
```

Comment:
Sorry if former comment https://github.com/openshift/openshift-docs/pull/74184#discussion_r1554849700 was not clear. It meant: instead of using the Shell variable format ISSUER_URL/$CLI_CLIENT_ID, let's use placehold text <issuer url> / <client id>